src/main/cpp/faultinj/faultinj.cu

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+#include <cudf/logger.hpp>
```

Comment:
The faultinj project does not link against any cudf libs, including the rapids_logger.  I think we need to add an explicit dependency on spdlog for this project fetching it if necessary.